lib/pages/home/home_direct_list_widget.dart

```diff
@@ -46,18 +46,44 @@ class HomeDirectListWidget extends StatelessWidget {
                 itemCount: directState.channels.length,
                 itemBuilder: (context, index) {
                   final channel = directState.channels[index];
-                  return HomeChannelTile(
-                    onHomeChannelTileClick: () =>
-                        NavigatorService.instance.navigate(
-                      channelId: channel.id,
-                      workspaceId: channel.workspaceId,
-                    ),
-                    title: channel.name,
-                    name: channel.lastMessage?.senderName,
-                    content: channel.lastMessage?.body,
-                    imageUrl: channel.icon,
-                    dateTime: channel.lastActivity,
-                    channelId: channel.id,
+
+                  return FutureBuilder(
+                    //  initialData: Account init,
+                    future: Get.find<ChannelsCubit>()
+                        .fetchMembers(channel: channel)
+                        .then((value) => value.first),
+                    builder: (context, snapshot) {
+                      if (snapshot.connectionState == ConnectionState.done) {
+                        return HomeChannelTile(
+                          onHomeChannelTileClick: () =>
+                              NavigatorService.instance.navigate(
+                            channelId: channel.id,
+                            workspaceId: channel.workspaceId,
+                          ),
+                          title: (snapshot.data as Account).fullName,
+                          name: channel.lastMessage?.senderName,
+                          content: channel.lastMessage?.body,
+                          imageUrl: (snapshot.data as Account).thumbnail,
```

Comment:
it's better if we start working solely on core-api-migration branch, because a lot of fields has changed (here thumbnail has been renamed to picture)